plaidml/tf_test.py

```diff
@@ -116,14 +126,28 @@ class TestTF(unittest.TestCase):
     """Tensorflow like operation tests."""
 
     @opTest([
-        [np.random.random((5, 60, 10, 3)), [1,3,4,1], [1,3,3,1], [1,1,1,1], "SAME"],
-        [np.random.random((5, 60, 10, 3)), [1,3,4,1], [1,3,3,1], [1,1,1,1], "VALID"],
-        [np.random.random((5, 60, 10, 3)), [1,4,3,1], [1,1,1,1], [1,1,1,1], "SAME"],
-        [np.random.random((5, 60, 10, 3)), [1,4,3,1], [1,1,1,1], [1,6,5,1], "SAME"],
-    ])
-    def testExtractImagePatches(self, b, images, ksizes, strides, rates=(1,1,1,1), padding="VALID"):
-        func = tensorflow.extract_image_patches if b == tf else plaidml.op.extract_image_patches
-        return [func(images, ksizes, strides, rates, padding)]
```

Comment:
This was probably accidentally removed ?

Reply:
Yup, restored in 0.6.1, also put tf_test into bazel so we run it in CI